src/components/Article.jsx

```diff
@@ -1 +1,30 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => (
+  <div className="box">
+    <h2 className="title" data-cy="title">
+      {title}
+    </h2>
+
+    <p className="subtitle">
+      <time dateTime={date.toISOString().slice(0, 10)} data-cy="date">
+        {date.toLocaleDateString('en-US', {
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric',
+        })}
```

Comment:
lets create variables for these date values, don't leave a lot of logic inside JSX